docs/MapColonies/Raster/Guides/raster-export.mdx

```diff
@@ -0,0 +1,247 @@
+                <mc:producerName>IDFMU</mc:producerName>
+                <mc:productBBox>-180,-90,180,90</mc:productBBox>
+                <mc:productId>ORTHOPHOTO_MOSAIC_BASE</mc:productId>
+                <mc:productName>אורתופוטו מתעדכן</mc:productName>
+                <mc:productType>OrthophotoBest</mc:productType>
+                <mc:productVersion>1.0</mc:productVersion>
+                <mc:region>World</mc:region>
+                <mc:maxResolutionDeg>0.000171661376953125</mc:maxResolutionDeg>
+                <mc:sensors>UNDEFINED</mc:sensors>
+                <mc:imagingTimeEndUTC>2020-05-21</mc:imagingTimeEndUTC>
+                <mc:imagingTimeBeginUTC>2020-05-21</mc:imagingTimeBeginUTC>
+                <mc:SRS>4326</mc:SRS>
+                <mc:SRSName>WGS84GEO</mc:SRSName>
+                <mc:transparency>OPAQUE</mc:transparency>
+                <mc:type>RECORD_RASTER</mc:type>
+                <mc:updateDateUTC>2022-02-13T13:03:07Z</mc:updateDateUTC>
+                <ows:BoundingBox crs="urn:x-ogc:def:crs:EPSG:6.11:4326" dimensions="2">
+                    <ows:LowerCorner>-180.0 90.0</ows:LowerCorner>
+                    <ows:UpperCorner>180.0 90.0</ows:UpperCorner>
+                </ows:BoundingBox>
+            </mc:MCRasterRecord>
+        </csw:SearchResults>
+    </csw:GetRecordsResponse>
+  ```
+</details>
+
+## Use Export Service (Step 2)
+Now you need to access export service Export and request export for the desired layer
+service Reference is [here](/docs/MapColonies/Raster/services/raster-export)
+
+POST Request
+``` json
+{
+  "catalogRecordID": "3fa85f64-5717-4562-b3fc-2c963f66afa6",
+  "domain": "RASTER",
+  "ROI": {
+    "type": "FeatureCollection",
+    "features": [
+      {
+        "type": "Feature",
+        "properties": {
+          "maxResolutionDeg": 0.0439453125,
+          "minResolutionDeg": 0.17578125
+        },
+        "geometry": {
+          "type": "Polygon",
+          "coordinates": [
+            [
+              [
+                34.82836896556114,
+                32.03918441418732
+              ],
+              [
+                34.81210152170695,
+                32.03918441418732
+              ],
+              [
+                34.81210152170695,
+                32.02539369969462
+              ],
+              [
+                34.82836896556114,
+                32.02539369969462
+              ],
+              [
+                34.82836896556114,
+                32.03918441418732
+              ]
+            ]
+          ]
+        }
+      }
+    ]
+  },
+  "artifactCRS": "4326",
+  "description": "export example",
+  "keywords": {
+    "foo": "kuku"
+  },
+  "parameters": {
+    "foo": "kuku"
+  },
+  "webhook": [
+    {
+      "events": [
+        "TASK_COMPLETED"
+      ],
+      "url": "https://webhook.com"
+    }
+  ]
+}
+```
+
+## Wait for callback (Step 3)
+Listen on "webhook url" that you provided for the response... :smile: enjoy
```

Comment:
And then what? Please explain what it looks like to get a response from the webhook and what to do next.